src/wallet/rpcwallet.cpp

```diff
@@ -3046,7 +3046,7 @@ static RPCHelpMan listunspent()
     };
 }
 
-void FundTransaction(CWallet* const pwallet, CMutableTransaction& tx, CAmount& fee_out, int& change_position, const UniValue& options, CCoinControl& coinControl)
+void FundTransaction(CWallet* const pwallet, CMutableTransaction& tx, CAmount& fee_out, int& change_position, const UniValue options, CCoinControl& coinControl, const UniValue& solving_data)
```

Comment:
Why is `options` losing its explicit ref typing?

Reply:
Oops. should be fixed.